spark_pipeline_framework/transformers/framework_fhir_meta_updater/v2/framework_fhir_meta_updater.py

```diff
@@ -0,0 +1,161 @@
+from typing import Dict, Any, Optional
+
+# noinspection PyProtectedMember
```

Comment:
I don't think, we need this

Reply:
base copied from v1. will check if it can be removed